docker-compose.yml

```diff
@@ -4,8 +4,13 @@ volumes:
     driver: local
     driver_opts:
       type: none
-      device: $DATA_DIR  # DATA_DIR is the host directory for the data. It has to be in the environment, e.g. in .env file
+      device: $DATA_DIR # DATA_DIR is the host directory for the data. It has to be in the environment, e.g. in .env file
       o: bind
+  tmp_packages: # do not change any detail of this volume
```

Comment:
I just copied the volume configuration from the other one for now.

Reply:
let's call it just `tmp_data`, there's no packages